Package.swift

```diff
@@ -29,11 +28,26 @@ let buildSettings: [CSetting] = [
         "-fdollars-in-identifiers",
         "-fno-common",
         "-fcf-runtime-abi=swift",
+        "-include",
+        "\(Context.packageDirectory)/Sources/CoreFoundation/internalInclude/CoreFoundation_Prefix.h",
         // /EHsc for Windows
     ]),
     .unsafeFlags(["-I/usr/lib/swift"], .when(platforms: [.linux, .android])) // dispatch
 ]
 
+#if os(macOS)
```

Comment:
nit: I think this can also be a `.when(platforms: [.macOS])` on the `.unsafeFlags` below to avoid the ifdefs here if we want